docs/api/text/umap_vis.rst

```diff
@@ -25,33 +26,34 @@ documents. The yellowbrick visualizer then plots the scatter plot,
 coloring by cluster or by class, or neither if a structural analysis is
 required.
 
-.. code:: ipython3
```

Comment:
This was causing a document build error - no pygments for ipython3 so updated it while I had the chance.